addons/account/tests/test_account_tax.py

```diff
@@ -31,6 +63,133 @@ def test_changing_tax_company(self):
         with self.assertRaises(UserError), self.cr.savepoint():
             self.company_data['default_tax_sale'].company_id = self.company_data_2['company']
 
+    def test_logging_of_tax_update_when_tax_is_used(self):
+        ''' Modifications of a used tax should be logged. '''
```

Comment:
```suggestion
        """ Modifications of a used tax should be logged. """
```
Several occurrences :wink: